dwave_networkx/generators/chimera.py

```diff
@@ -545,7 +545,7 @@ def __missing__(self, key):
 _chimera_coordinates_cache = __chimera_coordinates_cache_dict()
 
 def linear_to_chimera(r, m, n=None, t=None):
```

Comment:
Is this function not documented anywhere?

Reply:
Could you elaborate? `linear_to_chimera` does have a docstring.

Reply:
There's a ``linear_to_chimera``  method of ``chimera_coordinates`` defined on line 406 and a function ``linear_to_chimera`` defined on line 547, but I see only the first documented on line 40 of ``utilities.rst``